### PR TITLE
add export html to pdf utility

### PR DESCRIPTION
**Pull request by brandonwong368**
### Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/086c1b564dbf478faa649a8880e34b81?v=9d3f8002a0d1477ba395616dfc082af7&p=1be01de27e1c462bade88d67e612f917)

### Implementation description

- using html-pdf package
- placeholder variable for local path to html file or html string to convert into pdf
- downloads file to a designated path

### Steps to test

1. Added a random html file (test.html match the filename to the util) to the repository root
2. Created a new endpoint calling the function. Used postman to send a request
3. New pdf called test.pdf will be downloaded (the test html file and the pdf were deleted after testing)

### What should reviewers focus on?

-

### Checklist

-   [x] My PR name is descriptive and in imperative tense
-   [x] I have run the linter
-   [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by brandonwong368** [comment]
re opened #38 after joining Railway project

File: utils/htmlToPDF.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { create, CreateOptions } from "html-pdf";
+import { readFileSync } from "fs";
+
+/**
+ * converts hard coded html into a PDF and downloads the PDF locally
+ * @param {string} downloadPath download path of the generated pdf
+ */
+const htmlToPDF = (downloadPath: string): void => {
+    // path of the hard coded html or the html as a string
+    const htmlPath = "./test.html";
```
Comment (brandonwong368) [inline utils/htmlToPDF.ts]: Hard coded string will be provided

Comment (mahadzaryab1) [reply utils/htmlToPDF.ts]: If the hardcoded string will be provided then we should be taking in that string here

File: utils/htmlToPDF.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+/**
+ * converts hard coded html into a PDF and downloads the PDF locally
+ * @param {string} downloadPath download path of the generated pdf
+ * @param {string} volunteerName volunteer name
+ * @param {string} date date that the form will be filled out on, can we just pass in current date?
+ * @param {string} coordinatorName program coordinator name
+ * TODO: incorporate parameters into pdf after form html is provided
+ */
+const htmlToPDF = (
```
Comment (mahadzaryab1) [inline utils/htmlToPDF.ts:12]: how is this function going to be used? i don't think this should be doing any downloading but rather returning a PDF from the function - is that possible? the caller can do whatever they want with the file (store it, download it, etc.)

**Review by mahadzaryab1 — COMMENTED** [review]
Added a couple of comments/questions. Also, please look over the PR template - let's make the PR title more descriptive in imperative tense.

File: utils/htmlToPDF.ts (hunk 2 above)
Comment (feng-eric) [reply utils/htmlToPDF.ts:12]: I think this function will be used in the frontend for generating/downloading the PDF

**Comment by feng-eric** [comment]
Can you demo this to the team or to me some time in the next few days?

**Comment by MatoPlus** [comment]
closing stale PR

**Pull request closed without merging**